### PR TITLE
Add CDOGS marketing and other minor UI QoL fixes

### PR DESCRIPTION
**Pull request by jujaga**
# Description

TBD

[[SHOWCASE-430]](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-430)
[[SHOWCASE-452]](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-452)

## Types of changes

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by parc-jason** [comment]
Need to resolve conflicts before this can get built and tested.

File: app/frontend/src/App.vue [hunk 1]
```diff
@@ -17,10 +24,23 @@ import Footer from './components/Footer';
...
+  },
   components: {
     Header,
     NavigationBar,
     Footer
+  },
+  mounted() {
+    setTimeout(() => {
+      this.snack = !this.$router;
+      if (!this.$router && (this.$keycloak && !this.$keycloak.ready)) {
```
Comment (loneil) [inline app/frontend/src/App.vue]: All AND conditions, don't need the brackets

File: app/frontend/src/components/Examples.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              <th v-if="$vuetify.breakpoint.mdAndUp" class="text-left">Type</th>
+              <th class="text-left">Extension</th>
+              <th class="text-left">Template</th>
+              <th class="text-left">Contexts</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="(item, index) in examples" :key="`item-${index}`">
+              <td>{{ item.description }}</td>
+              <td v-if="$vuetify.breakpoint.mdAndUp">{{ item.type }}</td>
```
Comment (loneil) [inline app/frontend/src/components/Examples.vue]: Can probably change these to the display classes https://vuetifyjs.com/en/styles/display

Comment (jujaga) [reply app/frontend/src/components/Examples.vue]: I opted to explicitly leave this in for the table as it has some strange behavior when I tried to use the class based approach instead (alignment issues).

Comment (loneil) [reply app/frontend/src/components/Examples.vue]: Ok, no prob

File: app/frontend/src/App.vue (hunk 1 above)
Comment (jujaga) [reply app/frontend/src/App.vue]: Fixed

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wfezkf-tools.pathfinder.gov.bc.ca/dashboard?id=document-generation-showcase-pr-25

File: app/frontend/src/App.vue [hunk 3]
```diff
@@ -17,10 +24,23 @@ import Footer from './components/Footer';
...
+        console.error('Keycloak failed to initialize');
```
[the remark this answers is not in the document]
Comment (jujaga) [reply app/frontend/src/App.vue]: I've changed it after our discussion to `Something important failed while loading... :(`.

**Comment by jujaga** [comment]
>     * add some whitespace between the "try it out" button and the "github" button on the home page
>     * can you remove the underlying of text on those buttons?
>     * when you make the screen small the github button goes to just the icon, but there still seems to be plenty of room for the whole button. is this wrapping necessary?

Button whitespace added, the github collapsing behavior is intended by design, and the underline for those buttons have been removed.

**Pull request merged**